vocode/streaming/telephony/server/base.py

```diff
@@ -82,32 +102,28 @@ def __init__(
         for config in inbound_call_configs:
             self.router.add_api_route(
                 config.url,
-                self.create_inbound_route(
-                    agent_config=config.agent_config,
-                    twilio_config=config.twilio_config,
-                    transcriber_config=config.transcriber_config,
-                    synthesizer_config=config.synthesizer_config,
-                ),
+                self.create_inbound_route(inbound_call_config=config),
                 methods=["POST"],
             )
-            self.logger.info(
-                f"Set up inbound call TwiML at https://{base_url}{config.url}"
-            )
+        # vonage requires an events endpoint
+        self.router.add_api_route("/events", self.events, methods=["GET"])
```

Comment:
should we add this if it's twilio too?

Reply:
yes - a `TelephonyServer` can support both Vonage and Twilio calls, so we'll need to set it up in either case